amaranth-chrome-ext/src/CalorieLabel.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Possible calorie labels that can be given to a dish.
+ * @enum {number}
```

Comment:
Remove empty tag.

Reply:
[This is standard for JS enum docs](https://google.github.io/styleguide/jsguide.html#features-objects-enums). However, I can add some additional info if it would be useful. Perhaps explaining why the enum's value is a number?

Reply:
Fixed in https://github.com/googleinterns/amaranth/pull/44/commits/ec100b15fef71ddcc05d79b2feffaad66129055f